package.json

```
@@ -29,18 +29,12 @@
     "xcodebuild": "xcodebuild -quiet test -workspace tests/cordova-ios.xcworkspace -destination \"platform=iOS Simulator,name=iPhone 8\" CONFIGURATION_BUILD_DIR=\"`mktemp -d 2>/dev/null || mktemp -d -t 'cordova-ios'`\"",
     "preobjc-tests": "tests/scripts/killsim.js",
     "unit-tests": "jasmine --config=tests/spec/unit.json",
-    "eslint": "eslint . \"bin/**/!(*.*)\""
+    "lint": "eslint ."
```

```suggestion
    "lint": "eslint . \"bin/**/!(*.*)\""
```

The deleted part is needed to lint the JS "binaries" that have no `.js` extension.

---

I keep forgetting why we added this and removing it. I think this is now the second time.
Thanks for catching this again.